csv_datasets/ldbc/lists/edges-sf-003-full.txt

```diff
@@ -75,28 +75,28 @@ creationDate:DATETIME|:START_ID(Person)|:END_ID(Post)|:TYPE
 sf-003/dynamic/person_likes_post/part-00000-42cce85f-f27a-40d3-8c65-69f139ffd456-c000.csv.split_0
 sf-003/dynamic/person_likes_post/part-00000-42cce85f-f27a-40d3-8c65-69f139ffd456-c000.csv.split_1
 
-creationDate:DATETIME|:START_ID(Person)|:END_ID(Language)|:TYPE
+creationDate:IGNORE|:START_ID(Person)|:END_ID(Language)|:TYPE
 sf-003/dynamic/person_speaks_language/manually_transformed_2021_03_04.csv.split_0
 sf-003/dynamic/person_speaks_language/manually_transformed_2021_03_04.csv.split_1
 
-creationDate:DATETIME|:START_ID(Person)|:END_ID(Organisation)|classYear:LONG|:TYPE
+creationDate:IGNORE|:START_ID(Person)|:END_ID(Organisation)|classYear:LONG|:TYPE
 sf-003/dynamic/person_studyAt_organisation/part-00000-2156f71d-0672-4640-bb09-0fae5876122e-c000.csv.split_0
 sf-003/dynamic/person_studyAt_organisation/part-00000-2156f71d-0672-4640-bb09-0fae5876122e-c000.csv.split_1
 
 KATANA_DEFAULT_LABEL=WORK_AT
-creationDate:DATETIME|:START_ID(Person)|:END_ID(Organisation)|workFrom:LONG|:IGNORE
+creationDate:IGNORE|:START_ID(Person)|:END_ID(Organisation)|workFrom:LONG|:IGNORE
 sf-003/dynamic/person_workAt_organisation/part-00000-274143b8-ede7-4a41-b8c8-10fea2aa3056-c000.csv.split_0
 sf-003/dynamic/person_workAt_organisation/part-00000-274143b8-ede7-4a41-b8c8-10fea2aa3056-c000.csv.split_1
 
 KATANA_DEFAULT_LABEL=
-creationDate:DATETIME|:START_ID(Post)|:END_ID(Person)|:TYPE
+creationDate:IGNORE|:START_ID(Post)|:END_ID(Person)|:TYPE
 sf-003/dynamic/post_hasCreator_person/part-00000-10cb9ef3-dc6c-43e4-b6e7-a2179dcd810c-c000.csv.split_0
 sf-003/dynamic/post_hasCreator_person/part-00000-10cb9ef3-dc6c-43e4-b6e7-a2179dcd810c-c000.csv.split_1
 
-creationDate:DATETIME|:START_ID(Post)|:END_ID(Tag)|:TYPE
+creationDate:IGNORE|:START_ID(Post)|:END_ID(Tag)|:TYPE
 sf-003/dynamic/post_hasTag_tag/part-00000-8f918eb1-beed-428f-b68a-18f1c5e3d1d0-c000.csv.split_0
 sf-003/dynamic/post_hasTag_tag/part-00000-8f918eb1-beed-428f-b68a-18f1c5e3d1d0-c000.csv.split_1
 
-creationDate:DATETIME|:START_ID(Post)|:END_ID(Place)|:TYPE
+creationDate:IGNORE|:START_ID(Post)|:END_ID(Place)|:TYPE
 sf-003/dynamic/post_isLocatedIn_place/part-00000-d2a7a9ca-9711-48ec-b36d-3089c9a62f09-c000.csv.split_0
```

Comment:
this file looks good; same comments as in the previous one obviously, but matching